index.md

```diff
@@ -52,34 +50,15 @@ solver = dolfinx.fem.petsc.LinearProblem(
 solver.solve()
 ```
 
-The above code snippet also shows how to define a suitable finite element
-function space, using continuous piecewise quadratic vector-valued functions
-for the velocity and continuous piecewise linear functions for the pressure
-(Taylor-Hood). The computational domain and mesh are also easily created with
-[gmsh](https://gmsh.info/), here defined by three spheres immersed in a 3D
-channel.
+The above code snippet also shows how to define a suitable finite
+element function space, using continuous piecewise quadratic
+vector-valued functions for the velocity and continuous piecewise linear
+functions for the pressure (Taylor-Hood). The computational domain and
+mesh are also easily created with [gmsh](https://gmsh.info/), here
+defined by three spheres immersed in a 3D channel.
 
 ![Stokes example](/assets/img/stokesexample.png){: .image-center }
 
-```python
```

Comment:
One of the tests is failing because this code has gone.

You can remove this line and the comma on the line before and they should pass again: https://github.com/FEniCS/web/blob/426254ccdb69fabd658decfca6aa6fa350e69e10/_test/test_code.py#L23